Makefile

```diff
@@ -95,7 +95,7 @@ test:
 
 .PHONY: build_docs
 build_docs:
-	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) -b html5 -D $(DOCS_BUILD_DIR) $(DOCS_BUILD_TARGET)
+	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) -b html5 -D $(DOCS_BUILD_DIR) -o index.html $(DOCS_BUILD_TARGET)
```

Comment:
@anjannath you need to run the podman with `sudo` otherwise it is not able to get the network capability, I think it is fixed in later version of podman but package which is part of ubuntu repo might not the updated one. you can see the error from the CI.

```
Writing manifest to image destination
Storing signatures
ERRO[0014] could not find slirp4netns, the network namespace won't be configured: exec: "slirp4netns": executable file not found in $PATH 
```

Also I am not sure why this job is not failed even podman exited with an error (not sure if the exit code is correct here).

Reply:
This is strange, but it does create the `index.html` file not sure how, when i tested it pushed that file to my branch, instead of running using sudo let me try installing that package.